path_config.go

```diff
@@ -89,6 +89,13 @@ func pathConfig(b *jwtAuthBackend) *framework.Path {
 					Name: "Provider Config",
 				},
 			},
+			"namespace_in_state": {
+				Type:        framework.TypeBool,
+				Description: "Pass namespace in the OIDC state parameter instead of as a separate query parameter. With this setting, the allowed redirect URL(s) in Vault and on the provider side should not contain a namespace query parameter. This means only one redirect URL entry needs to be maintained on the provider side for all vault namespaces that will be authenticating against it. Defaults to true for new configs.",
+				DisplayAttrs: &framework.DisplayAttributes{
+					Name: "Namespace in OIDC state",
```

Comment:
Should we keep the value in `DisplayAttrs` for UI purposes? I am not sure if `Default` and `DisplayAttrs.Value` is required to match.

Reply:
Yep, I'll add the `DisplayAttrs` value back in. Added in 58a9841.

From my brief testing without the `DisplayAttrs` value, it shows up on the config edit page as unchecked. And if the user doesn't click on the field, it isn't sent when the config is submitted, which ends up defaulting to true on the backend. But it looks odd to submit a config with that value unchecked, only to find that the backend defaulted to having it checked.